src/components/common/FormBottomBar.tsx

```diff
@@ -90,7 +94,7 @@ export default function FormBottomBar({
         type='passive'
         className={classes.button}
       />
-      <div>
+      <div className={classes.rightButtonGroup}>
```

Comment:
thanks for fixing.. i see it broken in staging for other pages with a form bottom bar.
curious.. when/how did this break?

Reply:
It's likely i broke it when i was adding the ability to display more buttons in the form

Reply:
all cool, glad you caught it, thanks much 🙏